pkgs/tools/misc/SP800-90B_EntropyAssessment/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bzip2
+, libdivsufsort
+, jsoncpp
+, openssl
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "SP800-90B_EntropyAssessment";
+  version = "1.1.6";
+
+  src = fetchFromGitHub {
+    owner = "usnistgov";
+    repo = "SP800-90B_EntropyAssessment";
+    rev = "v${version}";
+    hash = "sha256-KZQ7kC0PbBkjLEQZIqYakQ91OvCxruhdfUwiRHtno3w=";
+  };
+
+  buildInputs = [ bzip2 libdivsufsort jsoncpp openssl mpfr ];
+
+  sourceRoot = "source/cpp";
+
+  makeFlags = lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+    "CROSS_COMPILE=${stdenv.cc.targetPrefix}"
+    "ARCH=${stdenv.hostPlatform.linuxArch}"
```

Comment:
Native aarch64 build fails
https://logs.ofborg.org/?key=nixos/nixpkgs.246568&attempt_id=fa519229-9a55-4eb7-8134-e150412ee5d3

```nix
  postPatch = ''
    substituteInPlace Makefile \
      --replace "-march=native" ""
  '' + lib.optionalString (!stdenv.hostPlatform.isx86) ''
    substituteInPlace Makefile \
      --replace "-msse2" ""
  '';
```
https://github.com/usnistgov/SP800-90B_EntropyAssessment/blob/1d8b0af9781eec03a862fa65256d68e107eee31c/cpp/Makefile#L7

`march=native` is impure and breaks running the program on cpus older than the builder cpu